templates/experiment.html

```diff
@@ -1,6 +1,6 @@
 <div class="abe-experiment-warning ampstart-card p1">
   <h4 class="mb1">Experimental Mode</h4>
-  <p>This example uses the following experimental feature{{#metadata.experiments.1}}s{{/metadata.experiments.1}}: <code class="amp-experiment-list">[{{#metadata.experiments}}<span>{{.}}</span>{{/metadata.experiments}}]</code>. Enable the experiment via the button below. Some components require the AMP Dev Channel to be enabled as well. <a href="https://www.ampproject.org/docs/reference/experimental.html">Learn more here</a>.</p>
+  <p>This example uses the following experimental feature{{#metadata.experiments.1}}s{{/metadata.experiments.1}}: <code class="amp-experiment-list">[{{metadata.experiments}}]</code>. Enable the experiment via the button below. Some components require the AMP Dev Channel to be enabled as well. <a href="https://www.ampproject.org/docs/reference/experimental.html">Learn more here</a>.</p>
```

Comment:
Nit: can you also add the 's' after 'Enable the experimentt{{#metadata.experiments.1}}s{{/metadata.experiments.1}}'.

Reply:
added